src/manage.c

```diff
@@ -3560,6 +3560,7 @@ handle_osp_scan (task_t task, report_t report, const char *scan_id)
   char *host, *ca_pub, *key_pub, *key_priv;
   int rc, port;
   scanner_t scanner;
+  gboolean started = FALSE;
```

Comment:
I know I'm nitpicking but could you init this in the group below for consistency.

Reply:
Done in 2d17b0139e45b59a70ca5aeca86412e15e86db79.